extensions/amp-consent/amp-consent.md

```diff
@@ -78,7 +78,11 @@ Example:
   "consents": {
     "my-consent": {
       "checkConsentHref": "https://example.com/api/show-consent",
-      "promptUI": "consent-ui"
+      "promptUI": "consent-ui",
+      "timeout": {
+        "seconds": 5,
+        "fallbackState": "rejected"/"unknown"
```

Comment:
only "rejected" will be supported.

Reply:
i thought i saw "unknown" during the code review.
if the field can only be "rejected", shall we take it out for now?